test/integration/test-pipe.js

```diff
@@ -36,7 +37,7 @@ var server = http.createServer(function(req, res) {
       var field = FIELDS.shift();
       assert.strictEqual(name, field.name);
       assert.strictEqual(file.name, path.basename(field.value.path));
-      assert.strictEqual(file.type, mime.lookup(file.name));
+      assert.strictEqual(file.type, mime.lookup(file.name) || defaultContentType);
```

Comment:
I'm not sure how to change this since this test submits multiple files, some with content-types and some without.